armi/bookkeeping/db/tests/test_database3.py

```diff
@@ -128,6 +128,71 @@ def _compareRoundTrip(self, data):
         roundTrip = database3.unpackSpecialData(packed, attrs, "testing")
         self._compareArrays(data, roundTrip)
 
+    def test_prepRestartRun(self):
+        # This test is based on the armiRun.yaml case that is loaded during the `setUp`
+        # above. In that cs, `reloadDBName` is set to 'reloadingDB.h5', `startCycle` = 1,
+        # and `startNode` = 2. The nonexistent 'reloadingDB.h5' must first be
+        # created here for this test.
```

Comment:
Can we just make this a multi-line docstring using `"""`?

Thanks!

Reply:
Done.